src/app/core/user-module/user.routes.ts

```diff
@@ -45,14 +45,8 @@ export const USER_ROUTES = {
   UPDATE_USER(user: string) {
     return `${environment.apiURL}/users/${encodeURIComponent(user)}`;
   },
-  /**
-   * Request to retrieve a user's profile
-   * @method GET
-   * @auth required
-   * @param username the username of the user
-   * @returns the user's profile
-   */
-  GET_USER_PROFILE(username: string) {
-    return `${environment.apiURL}/users/${encodeURIComponent(username)}/profile`;
```

Comment:
why was this removed?

Reply:
it's not used anywhere and i haven't seen any mention of it on the analysis docs